test/banners/mobile/components/BannerCtrl.spec.ts

```diff
@@ -75,15 +74,6 @@ describe( 'BannerCtrl.vue', () => {
 		vi.useRealTimers();
 	} );
 
-	describe( 'Content', () => {
```

Comment:
These tests make sure the animated `visitorsVsDonors` sentence is in the banners, can you put them back?

Please also add `expectHidesAnimatedVisitorsVsDonorsSentenceInMessage` and `expectHidesAnimatedVisitorsVsDonorsSentenceInSlideShow` to the data provider array.